playground/seq/Seq.ApiService/Program.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+
+var builder = WebApplication.CreateBuilder(args);
+
+Seq.ServiceDefaults.Extensions.AddServiceDefaults(builder);
```

Comment:
Is this what you expect the code to look like?

Reply:
No. I've changed it.